apps/web/src/auth/CommunityAuthContextProvider.tsx

```diff
@@ -0,0 +1,24 @@
+import { createContext } from 'react';
```

Comment:
Can we please move these under web/src/providers and rename to CommunityAuthProvider like all the other providers?